src/api/app/models/comment.rb

```diff
@@ -49,17 +47,27 @@ def create_notification(params = {})
     when 'Package'
       params[:package] = commentable.name
       params[:project] = commentable.project.name
-      # call the action
-      Event::CommentForPackage.create(params)
     when 'Project'
       params[:project] = commentable.name
-      # call the action
-      Event::CommentForProject.create(params)
     when 'BsRequest'
       params = commentable.notify_parameters(params)
       # params[:id] got overwritten with notify_parameters. We need to assign it again
       params[:id] = id
-      # call the action
+    end
+    params
+  end
+
+  private
+
+  def create_notification(params = {})
```

Comment:
This is creating events, not notifications, right?

Reply:
Done!